### DISCUSSION TITLE
Will superwerker work with an existing LZ and w/o the root email feature

### DISCUSSION THREAD

**Discussion by spurohit42** [superwerker/superwerker#305] [Q&A] [answered]
I have provisioned a brand new LZ and upgraded it to 2.9. Will superwerker work with such an LZ? There are no member accounts other than 1 in sandbox OU created to test the no-default-vpc settings. 

Also can it work w/o the root email feature? because I have established my own email convention, and the logarcgibe and security accounts are following that convention,  deliberately using email outside of AWS for root emails. Incidentally I dont fully agree that the email should not be a distribution list. I do agree that you need a convention and that the emial must be secure and if its a distribution list it must be controllable.

**Comment by s0enke** [comment, marked as answer]
Hi [spurohit42](https://github.com/spurohit42),

Thanks for the question. Currently, the `rootmail` feature is mandatory (more explanation in [this discussion](https://github.com/superwerker/superwerker/discussions/283), and installing superwerker into an AWS account with existing Control Tower would fail because a [`SetupLandingZone` EventBridge event](https://github.com/superwerker/superwerker/blob/b9551dd649241f761b562d97bc04132f9467920a/templates/control-tower.yaml#L140-L159) would have never been triggered. This can possibly be worked around by also listening to `UpdateLandingZone` events.
